modules/manage/pages/schema-reg/schema-reg-overview.adoc

```diff
@@ -33,7 +33,39 @@ See the xref:reference:cluster-properties.adoc#kafka_nodelete_topics[kafka_nodel
 
 endif::[]
 
-Redpanda Schema Registry uses the default port 8081. 
+Redpanda Schema Registry uses the default port 8081.
+
+== Serialization and deserialization
```

Comment:
@rockwotj @mattschumpert Does this subheading make sense or does it need to specifically mention the wire format?

Reply:
Calling it the wire format makes sense, because you can serialize/deserialize without it by having another mechanism to map a topic record to a schema: static mapping of topic to latest schema in your producer/consumer, communicating the schema ID using some other out of band mechanism (message header, control messages, etc).

Generally this is the "eco system standard" way of doing it.